### PR TITLE
OC-8689: Remove xdarklaunch couchdb switching

### PR DESCRIPTION
**Pull request by hosh**
WIP

This has some comments from temp/ruby-depsolver

Required: https://github.com/opscode/chef_db/pull/36

### CODE REVIEW

**Comment by hosh** [comment]
Ping: @seth @sdelano @manderson26 @marcparadise @oferrigni

**Comment by seth** [comment]
Let's get this rebased on master and get a build put together. Looks like it is covering the bits we are expecting.

**Comment by sdelano** [comment]
I rebased off master and force-pushed. Now we're able to get rid of the `temp/ruby-depsolver` mess and have a clear look at the PR.

File: src/chef_wm_search.erl [hunk 1]
```diff
@@ -114,7 +114,7 @@ resource_exists_message(org_not_found, Org) ->
 to_json(Req, #base_state{chef_db_context = DbContext,
                          resource_state = SearchState,
                          organization_name = OrgName,
-                         darklaunch = Darklaunch,
+                         batch_size = BatchSize,
```
Comment (seth) [inline src/chef_wm_search.erl]: This doesn't look right. Maybe missed during the rebase? `batch_size` isn't part of the `#base_state{}` record any more.

Comment (sdelano) [inline src/chef_wm_search.erl]: @seth Yes I missed this in rebase. I fixed it yesterday.

**Pull request merged**